courses/machine_learning/production_ml_systems/1_training_at_scale.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>typos: notebo -&gt; notebook and CCS -&gt; GCS</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/584/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>